Dockerfile

```
@@ -0,0 +1,27 @@
+ARG PYTHON_VERSION=3.10
+
+FROM python:${PYTHON_VERSION} AS base
+
+WORKDIR /app
+
+RUN pip3 install django
+
+#RUN python manage.py migrate
+
+COPY . .
+
+FROM python:${PYTHON_VERSION}-slim as run
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=base /app .
+
+RUN pip install -r requirements.txt
```

Add pip update before installation
```suggestion
RUN pip install --upgrade pip && \
    pip install -r requirements.txt
```